tests/dash/packets.py

```diff
@@ -103,13 +103,13 @@ def inbound_pl_packets(config, inner_packet_type='udp', vxlan_udp_dport=4789):
         ip_src=pl.PE_PA,
         ip_dst=pl.APPLIANCE_VIP,
         gre_key_present=True,
-        gre_key=int(pl.VNET1_VNI) << 8,
+        gre_key=int(pl.ENCAP_VNI) << 8,
```

Comment:
For all the changes in this file, can you set the values at runtime based on the testbed platform being used? The current hard-coded values are required for this test to pass on other platforms, runtime determination would allow more flexibility